### PR TITLE
Do not require crowbar-pacemaker cookbook to run when we configure HA in openstack

### PR DESCRIPTION
**Pull request by vuntz**
We make it possible to not run the crowbar-pacemaker cookbook when we run some OpenStack cookbook that use HA and require some specific setup (for drbd or haproxy, for instance).

This depends on and needs to be merged at the same time as: https://github.com/crowbar/crowbar-openstack/pull/635

Note: this includes a patch similar to https://github.com/crowbar/crowbar-ha/pull/116

### CODE REVIEW

**Comment by vuntz** [comment]
Re-triggered the CI with https://github.com/crowbar/crowbar-openstack/pull/635

**Comment by vuntz** [comment]
Gating is green when running with https://github.com/crowbar/crowbar-openstack/pull/635

File: chef/cookbooks/crowbar-pacemaker/recipes/openstack.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#
+# Author:: Vincent Untz
+# Cookbook Name:: crowbar-pacemaker
+# Recipe:: openstack
+#
+# Copyright 2016, SUSE
```
Comment (nicolasbock) [inline chef/cookbooks/crowbar-pacemaker/recipes/openstack.rb]: Could you update the year?

File: chef/cookbooks/crowbar-pacemaker/recipes/attributes.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#
+# Author:: Vincent Untz
+# Cookbook Name:: crowbar-pacemaker
+# Recipe:: attributes
+#
+# Copyright 2016, SUSE
```
Comment (nicolasbock) [inline chef/cookbooks/crowbar-pacemaker/recipes/attributes.rb]: Year.

File: chef/cookbooks/crowbar-pacemaker/recipes/remote_attributes.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#
+# Author:: Vincent Untz
+# Cookbook Name:: crowbar-pacemaker
+# Recipe:: attributes
+#
+# Copyright 2016, SUSE
```
Comment (nicolasbock) [inline chef/cookbooks/crowbar-pacemaker/recipes/remote_attributes.rb]: Year.

File: chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb [hunk 4]
```diff
@@ -0,0 +1,71 @@
+#
+# Cookbook Name:: crowbar-pacemaker
+# Recipe:: drbd
+#
+# Copyright 2014, SUSE
```
Comment (nicolasbock) [inline chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb]: Should we also update this year? The code is in a new file after all :smile:

File: chef/cookbooks/crowbar-pacemaker/recipes/openstack.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/crowbar-pacemaker/recipes/openstack.rb]: See the date of the commit ;-)

File: chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb (hunk 4 above)
Comment (vuntz) [inline chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb]: Well, it's a new file, but the code is coming from the old file, so...

File: chef/cookbooks/crowbar-pacemaker/recipes/openstack.rb (hunk 1 above)
Comment (nicolasbock) [inline chef/cookbooks/crowbar-pacemaker/recipes/openstack.rb]: Ok, fair enough :)

File: chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb (hunk 4 above)
Comment (nicolasbock) [inline chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb]: 2014 just looked sooo old :wink: But ok, I am convinced...

File: chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb [hunk 5]
```diff
@@ -62,5 +62,10 @@
 
 include_recipe "drbd::install"
 include_recipe "drbd::config"
-# this will start drbd
-include_recipe "drbd::default"
+
+# Disable drbd as it should never be started on boot (pacemaker should do i)
```
Comment (toabctl) [inline chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb]: typo, ``do it```

Comment (vuntz) [inline chef/cookbooks/crowbar-pacemaker/recipes/drbd_setup.rb]: I'll fix it, and merge, thanks!

**Pull request merged**